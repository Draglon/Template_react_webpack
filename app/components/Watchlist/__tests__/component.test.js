import React from 'react';

import WatchlistComponent from '../component';

describe('<WatchlistComponent />', () => {
  let component;
  const props = {
    watchlist: {
      results: [1, 2, 3],
      page: 1,
      totalPages: 100,
    },
    page: () => ({ page: 1 }),
    modalParams: { watchlist: false },
  };

  it('should match snapshot', () => {
    component = shallow(<WatchlistComponent {...props} />);
    expect(component).toMatchSnapshot();
  });
});
