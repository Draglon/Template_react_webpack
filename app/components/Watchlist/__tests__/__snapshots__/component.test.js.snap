// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<WatchlistComponent /> should match snapshot 1`] = `
<Fragment>
  <Row
    gutter={0}
  >
    <Col
      offset={2}
      span={20}
    >
      <div
        className="top-margin"
      >
        <Title>
          Watchlist
        </Title>
      </div>
    </Col>
  </Row>
  <MovieListComponent
    modalParams={
      Object {
        "watchlist": false,
      }
    }
    movies={
      Array [
        1,
        2,
        3,
      ]
    }
  />
  <PaginationComponent
    currentPage={1}
    page={[Function]}
    totalPages={100}
  />
</Fragment>
`;
